### PR TITLE
#1926: Added My Name to credits

### LINKED ISSUE #1926: Benjamin Kataoka - Credits
### Description

Giving myself credit for the amazing developing work I will be doing at NER!

### Acceptance Criteria

See my name in my favorite color on the credits page

### Proposed Solution

Add my name and favorite color to the "names" array in the CreditsPage.tsx file

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by BenjiKataoka**
## Changes

Changed the credits page to include my name

## Notes
N/A

## Screenshots
Screenshot 2024-01-24 at 2 16 37 PM
>

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #1926

### CODE REVIEW

**Review by martin0he — APPROVED** [review]
make sure to tick off everything on the checklist before pinging for a review. also, make sure you only ping **tech leads or higher** for a review. otherwise, great job keep it up

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+DATABASE_URL="postgresql://postgres:docker@localhost:5432/nerpm?schema=public"
```
Comment (RChandler234) [inline .env:1]: don't include this file in the PR

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Comment by BenjiKataoka** [comment]
I hope I fixed everything wrong with my ticket

**Comment by martin0he** [comment]
good job fixing the files, now all u have to do is fix the merge conflict. this means that your current working branch is out of date with the main develop branch. in your branch cmd, run git pull develop to update your current working branch.

**Comment by RChandler234** [comment]
> good job fixing the files, now all u have to do is fix the merge conflict. this means that your current working branch is out of date with the main develop branch. in your branch cmd, run git pull develop to update your current working branch.

I can just take care of the merge conflict

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**